internal/streamingnode/server/resource/idalloc/allocator.go

```diff
@@ -47,13 +47,11 @@ func NewIDAllocator(rc types.RootCoordClient) Allocator {
 }
 
 type remoteBatchAllocator interface {
-	batchAllocate(ctx context.Context, count uint32) (uint64, int, error)
+	batchAllocate(count uint32) (int64, int, error)
```

Comment:
Keep ctx in parameter is better to control the rpc timeout and easier to debug.
So wrap the original interface is better?

Reply:
ok